frontend/src/components/Plan/Plan.module.css

```diff
@@ -0,0 +1,100 @@
+.topContainer {
+  display: grid;
+  padding: 0;
+  grid-template-columns: 0.5fr repeat(5, 1fr);
+  margin: 1rem;
+  margin-right: 30px;
+}
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.planBtn {
+  background: none;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
+
+.burgerBtn {
+  width: 0;
+}
+
+.burgerBtnWrapper {
+  display: flex;
+  justify-content: start;
+}
+
+.collectedContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 19rem;
+  height: 5.5rem;
+  flex-flow: wrap;
+  background-color: #F5f5f7;
+  border-radius: 10px;
+  box-shadow: 5px 5px 15px #d7d7d7;
+  grid-column: 7;
+  padding: 0.5rem;
+  gap: 0.5rem;
+}
+
+.specialPersonWrapper {
+  display: flex;
+  text-align: left;
+  grid-column: 2/5;
+  height: 100%;
+  align-items: center;
+}
+
+.specialPerson {
+  margin: 0;
+}
+
+#collectedH4 {
+  flex-basis: 100%;
+  text-align: center;
+  text-align: center;
+  align-items: center;
+  margin: 0;
+  color: var(--black);
+}
+
+@media (min-width: 800px) {
+  #collectedH4 {
```

Comment:
im not sure off the top of my head what happens when this media tag is applied, do these properties overwrite the ones before? does this block wholly replace the previous one? i think standard practice is to conditionally apply CSS classes based on these media queries, rather than use the same name

Reply:
im not truly a css expert but from what I've seen at least, generally you don't have CSS classes defined in two places like you do here. you have a `#collectedH4` defined within this media tag and outside it. generally I see CSS written as like `plan-card-xl` and `plan-card-xs` to denote classes used at different sizes